### ISSUE TITLE
Main Coursework - Question 3A

### ISSUE THREAD

**Issue by SamSJackson** [jakelever/textasdata_bugbounty#62]
This may not be an issue, very well could have misunderstood. 

In the coursework, for comparing classifiers, we are told to train on the training set and evaluate on the validation set.
![image](https://user-images.githubusercontent.com/86316114/222976106-068eafab-2eb3-4d19-8aee-0d992805cf21.png)

However, in the paragraph below, which gives some guidance on discussing the metrics, it is a little more vague.
Are we being asked to evaluate on the validation set **and** the training set?

![image](https://user-images.githubusercontent.com/86316114/222976168-0351b248-5abe-4e5b-b22f-681b6781c5e1.png)

**Comment by jakelever** [comment]
Hi Sam, the main part of the question at the top gives the overall goal while the subpart gives some more details. The main goal is evaluating on the validation set. However you are asked to evaluate on the training set as well (as we did in lab) and discuss those performance metrics. So yes, you need to do both to answer the question.
